src/connectedk8s/setup.py

```diff
@@ -17,7 +17,7 @@
 # TODO: Confirm this is the right version number you want and it matches your
 # HISTORY.rst entry.
 
-VERSION = '1.6.2'
+VERSION = '1.6.3'
```

Comment:
Please note that the CLI team has defined a unified new extension version upgrade rule, please take a look https://github.com/Azure/azure-cli/blob/dev/doc/extensions/versioning_guidelines.md#cli-extension-version-transition-table

Reply:
Thank you for the heads up! Confirming that this is indeed a bug fix and 1.6.3 stands as the appropriate next version